notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>URIs -&gt; Cloud Storage locations</p><p> </p><p>The (You can ...) should be in note form:L</p><p> </p><p>*Note:* You can ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>s/"Cloud Storage Locations"/URIs: Ack.</p><p> </p><p>Note: done</p>